packages/managed-oracle-v2/src/mappings/optimisticOracleV2.ts

```diff
@@ -17,6 +18,18 @@ let network = dataSource.network();
 let isMainnet = network == "mainnet";
 let isGoerli = network == "goerli";
 
+function getCustomBond(requester: Address, identifier: Bytes, ancillaryData: Bytes): CustomBond | null {
+  const managedRequestId = getManagedRequestId(requester, identifier, ancillaryData).toHex();
```

Comment:
Should we use toHexString() for consistency with other ids?
```suggestion
  const managedRequestId = getManagedRequestId(requester, identifier, ancillaryData).toHexString();
```